README.md

```diff
@@ -1038,3 +1038,10 @@ generateQueryEmbeddings().then((embeddingsResponse) => {
 
 // << Send query to Pinecone to retrieve similar documents >>
 ```
+
+## Testing
+
+All unit, integration, and end-to-end testing takes place automatically in CI and is configured using Github actions
+and workflows, located in the `.github` directory of this repo.
+
+To run tests locally, see [CONTRIBUTING.md](CONTRIBUTING.md) for instructions.
```

Comment:
I think this can probably be cut. Maybe leave a note directing people to CONTRIBUTING.md for the development readme. 

End-users trying to get their app built (the main audience for this readme) shouldn't ever need to run our tests. Also, explaining about the `.github` directory doesn't seem like the right level of granularity. Github Actions is in such widespread use on github that I think it feels like a waste of the reader's time to explain here. It's kind of like how we don't write a lot to explain other conventional aspects, like how we track module dependencies in our `package.json` file.